src/types.py

```diff
@@ -0,0 +1,62 @@
+from numpy import dtype, array, recarray, uint8, ndarray
+from dataclasses import dataclass
+from numpy.typing import ArrayLike
+
+#FIXME: import flags from argparse for colormode
+
+# RGBA Color
+@dataclass
+class RGBA():
+    r:uint8
+    g:uint8
+    b:uint8
+    a:uint8
```

Comment:
alpha is generally from 0 to 1, but we can create a conversion function. It's nice to have them be all the same data type

Reply:
I changed alpha to be represented by a float32 -- depending on the image library we use we may need to update this

Reply:
matplotlib uses 0 to 1 for all 4 of them. OpenCV uses 0 to 255 for RGB and 0 to 1 for alpha

Reply:
https://docs.opencv.org/4.x/d5/dc4/tutorial_adding_images.html 
https://docs.opencv.org/3.4/d3/d96/tutorial_basic_geometric_drawing.html 

There's not a straight forward way to modify alpha in opencv as far as I've looked.